optaweb-employee-rostering-backend/src/main/java/org/optaweb/employeerostering/domain/rotation/TimeBucket.java

```diff
@@ -65,7 +65,7 @@
                     @JoinColumn(name = "day_id",
                             referencedColumnName = "id",
                             foreignKey = @ForeignKey(name = "DAY_FK",
-                                    foreignKeyDefinition = "FOREIGN KEY (day_id) references public.time_bucket (id)" +
+                                    foreignKeyDefinition = "FOREIGN KEY (day_id) references public.timebucket (id)" +
```

Comment:
Any reason for this?
If possible, I'd like to keep as many things as possible as similar as possible.

Reply:
Differences in hibernate table name. Spring Boot uses "Time_Bucket", Quarkus uses "TimeBucket"

Reply:
(The alternative is to rename the table name via the `@Entity` annotation)

Reply:
I would.

Reply:
Done for backwards compatibility with previous databases so no migration would be needed.